### ISSUE TITLE
Inconsistent GDS guidance on ALT text

### ISSUE THREAD

**Issue by abbott567** [alphagov/govuk-design-system#1887]
In the [Design System guidance on images](https://design-system.service.gov.uk/styles/images/#alternative-text) it covers alt text. The examples show you an example of using the alt attribute to describe the image. For example:

```html
<img src="larry-the-cat.jpg" alt="Larry the cat, Chief Mouser to the Cabinet Office, sitting on a meeting table wearing a Union Jack bowtie.">
```
This guidance does not align with the recently [updated GDS guidance on alt text](https://www.gov.uk/guidance/content-design/images#alt-text) in the Content Design guidance on GOV.UK. In this guidance it says best-practice is to use an empty alt attribute on all images and describe the image in the body copy. For example: 

> Describe what’s happening in the image in the body text and leave the ‘Alt text’ field empty. This means the description is available to everyone.

The GDS guidance on alt text should be aligned so other teams and departments do not get confused.

**Comment by EoinShaughnessy** [comment]
@abbott567 Thanks for flagging this! Will update our guidance to make sure it's aligned.

**Comment by joelanman** [comment]
I think we need to clarify our guidance, but just wondering if the content guidance for GOV.UK Publisher might be different to other services. For example if you use an icon in your service, it might help to add alt text. A different context to the content published on GOV.UK publishing.

**Comment by abbott567** [comment]
Yeah good point @joelanman - I think the example in the design system at the moment with Larry the cat could be aligned, and then additional guidance around icons would make it much more specific.

**Comment by joelanman** [comment]
just a quick update to say this might take a bit longer as we discuss more widely with the accessibility community at GDS. The two areas of guidance (GOV.UK Publishing and the Design System) are run by different teams

**Comment by joelanman** [comment]
We met with the accessibility community and confirmed that the [Content Guidance](https://www.gov.uk/guidance/content-design/images#alt-text) in the original post is specifically for people using GOV.UK Publisher. We'll continue to look into clarifying our guidance in the Design System.

We're looking for examples of images in services (any kind of image) so if anyone knows any, please add to the thread!

**Comment by jbuller** [comment]
A common example is the rendering of an uploaded photo, either of a person or a document etc. 
In these cases the alt needs to be there to reassure that the upload worked but be generic as we can't know what it depicts.

**Comment by joelanman** [comment]
@jbuller thats a great example thanks!

**Comment by EoinShaughnessy** [comment]
A DVSA friend mentioned the image on their [Tyre defects and damage](https://www.gov.uk/government/publications/tyre-defects-and-damage-hgvs-buses-and-trailers/tyre-defects-and-damage-hgvs-buses-and-trailers) page.

It shows a metal probe against a red cloth(?) background, and its alt text is "Tyre probe".

**Comment by EoinShaughnessy** [comment]
Some more examples, provided by the Accessibility team, from [Web Accessibility Tutorials content on informative images](https://www.w3.org/WAI/tutorials/images/informative/):<img width="938" alt="Screenshot 2021-10-20 at 15 50 16" src="https://user-images.githubusercontent.com/72507742/138116970-3e2250c3-caa6-4b07-93d5-3272f06ad964.png"><img width="951" alt="Screenshot 2021-10-20 at 15 50 44" src="https://user-images.githubusercontent.com/72507742/138117014-5565f840-44ae-4d6c-b776-509c4f770306.png">

**Comment by EoinShaughnessy** [comment]
[DfE marketing/campaign site with images and alt text](https://getintoteaching.education.gov.uk/funding-your-training)

**Comment by EoinShaughnessy** [comment]
This issue was originally about a discrepancy between Design System guidance and 'publishers' guidance. Design System advises using the `alt` attribute. 'Publishers' advises leaving this attribute empty. We wondered whether leaving it empty might exclude users of screen readers, because then they would be left unaware there was an image on a page. However, there have been no reports of users experiencing this issue.

As we looked into alt text more, the issue's scope widened. We realised that Design System's guidance possibly needed revision. For example, because its current image is an unlikely example of what a service would actually use. However, our research into images on government services failed to turn up any appropriate images we could use instead.

Following discussion with the Accessibility Team's content designer, we've agreed for now to update our respective pieces of guidance to specify who the guidance is for. Hopefully that will address the original issue. I'll also update the Design System guidance with a link to the 'publishers' guidance. Hopefully this will help redirect any users who accidentally land in the Design System.

When a new accessibility specialist joins and has bandwidth to define the best way to use alt text, we can also revisit the Design System guidance. To make sure we do this, I'll create a new issue and link it to this one, so we've preserved everyone's feedback - for example, on the different techniques for adding images.

**Comment by abbott567** [comment]
Thanks for the update @EoinShaughnessy - We've just changed the [alt text guidance in our accessibility manual](https://accessibility-manual.dwp.gov.uk/guidance-for-your-job-role/content-designer#alternative-text-for-non-text-content) to also differentiate between GOV.UK Publishing and Service Teams, based on a content review by GDS. So it sounds like it should all hopefully align with those changes. Thanks for persevering with this issue!

**Comment by EoinShaughnessy** [comment]
Cheers @abbott567, and thanks for the heads up about the manual update!
